hermes-common/src/main/java/pl/allegro/tech/hermes/common/config/Configs.java

```diff
@@ -179,6 +179,13 @@
     FRONTEND_STARTUP_TOPIC_SCHEMA_LOADING_RETRY_COUNT("frontend.startup.topic.schema.loading.retry.count", 3),
     FRONTEND_STARTUP_TOPIC_SCHEMA_LOADING_THREAD_POOL_SIZE("frontend.startup.topic.schema.loading.thread.pool.size", 16),
 
+    FRONTEND_PUBLISHING_STARTUP_VALIDATION_ENABLED("frontend.publishing.startup.validation.enabled", false),
+    FRONTEND_PUBLISHING_STARTUP_VALIDATION_TOPIC_NAME("frontend.publishing.startup.validation.topic.name", "pl.allegro.tech.skylab.test"),
+    FRONTEND_PUBLISHING_STARTUP_VALIDATION_TIMEOUT_MS("frontend.publishing.startup.validation.timeout.ms", 24 * 60 * 60 * 1000L),
+    FRONTEND_PUBLISHING_STARTUP_VALIDATION_MESSAGES_COUNT("frontend.publishing.startup.validation.messages.count", 100),
+    FRONTEND_PUBLISHING_STARTUP_VALIDATION_THREAD_POOL_SIZE("frontend.publishing.startup.validation.thread.pool.size", 10),
```

Comment:
Why this pool is so big ? We probably need only one publishing thread ? Maybe this config can be harcoded just like: https://docs.oracle.com/javase/7/docs/api/java/util/concurrent/Executors.html#newSingleThreadExecutor()

Reply:
I am thinking to change it by setting default 1 in config but keep option to publish it with Executors.newFixedThreadPool() to make it more flexible, can we go with this option? Because in this case we are free to increase number of messages in validation